assets/src/block-templates/issues/template.js

```diff
@@ -45,7 +45,7 @@ const item = ['core/group', {
 
 const template = ({
   backgroundColor = isNewIdentity ? 'beige-100' : 'grey-05',
-  titlePlaceholder = __('Enter title', 'planet4-blocks-backend'),
+  title = '',
```

Comment:
This is to be consistent with other templates, the idea is that this could potentially be used as the final title so it's not a placeholder